public/utils/helpers.js

```diff
@@ -46,3 +47,17 @@ export const inputLimitText = (
     </EuiText>
   );
 };
+
+export async function getAlerts({ params, httpClient, notifications, location, history }) {
```

Comment:
Can we add some documentation to explain the different parameters?
For example I cant easily tell what notifications, location, and history are